### PR TITLE
Add Maps to Dashboard

### LINKED ISSUE #2656: [DOC] Add Maps to dashboard
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
In 2.6, Maps users will be able to add maps into dashboards.
Open saved maps from Stack Management -> Saved Objects -> Open Maps

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
https://github.com/opensearch-project/dashboards-maps/issues/214
https://github.com/opensearch-project/dashboards-maps/issues/212

### PR DESCRIPTION
**Pull request by carolxob**
### Description
Describes how to add Maps to the Dashboard.

### Issues Resolved
Closes #2656

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
@carolxob - is this feature only for 2.6? If so, we don't backport unless they are backporting the functionality.

File: _dashboards/maps-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* Adjust and drag the map panel to change its size
+* Manage geodata queries in the Maps panel by using the Dashboard query bar state
+* Edit maps
+* Use the Maps editing page to save and return to the main Dashboard page
+* Create new maps directly from your dashboard
+
+
+## Getting started
+
+To get started, perform the following steps:
```
Comment (junqiu-lei) [inline _dashboards/maps-dashboard.md:24]: This part may need update, the entrypoint is Dashboard instead of Visualize

**Comment by carolxob** [comment]
@junqiu-lei This is ready for tech review. Thanks!

File: _dashboards/maps-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: Adding Maps to your Dashboard
+grand_parent: Building data visualizations
+parent: Using coordinate and region maps
```
Comment (junqiu-lei) [inline _dashboards/maps-dashboard.md:5]: This feature isn't related to coordinate and region maps, I think it belongs to `_dashboards/maps-plugin.md`

**Comment by junqiu-lei** [comment]
I saw this PR tagged with back-port to 2.0 - 2.5, this new feature is only available since 2.6, not sure if we need backport them. Thanks!

**Comment by kolchfa-aws** [comment]
@VijayanB @junqiu-lei Could you tech review this PR please?

**Review by VijayanB — APPROVED** [review]

**Review by junqiu-lei — APPROVED** [review]

File: _dashboards/visualize/maps-plugin.md [hunk 3]
```diff
@@ -1,22 +1,35 @@
...
-# Using the Maps plugin
+# Using maps 
 
 With OpenSearch Dashboards, you can create maps to visualize your geographical data. OpenSearch lets you construct map visualizations with multiple layers, combining data across different indexes. You can build each layer from a different index pattern. Additionally, you can configure maps to show specific data at different zoom levels. OpenSearch maps are powered by the OpenSearch maps service, which uses vector tiles to render maps. 
 
-## Getting started
+## Creating a new map
 
-To get started, perform the following steps:
+You can create a new map from the **Maps** or **Visualize** workflows by performing the following steps:
```
Comment (cwillum) [inline _dashboards/visualize/maps-plugin.md:17]: Feel free to disregard this. I'm looking at the great screenshots you added. Are the "workflows" actual UI elements? If they're not items you can actually see labeled or interact with, I'm not sure we need to make them bold face. But I'm not in the UI. So move on if this comment isn't valid.

File: _dashboards/visualize/maps-plugin.md [hunk 4]
```diff
@@ -1,22 +1,35 @@
...
+
+- To create a new map from the **Visualize** workflow, perform the following steps:
+
+  1. On the top menu bar, go to **OpenSearch Dashboards > Visualize**.
+  1. Select the **Create visualization** button.
+  1. In the **New Visualization** dialog, select **Maps**.
+
+You can now see the default OpenSearch basemap.
+
+To examine the **Default map** layer configuration, in the **Layers** panel on the upper left of the map, select **Default map**, as shown in the following image.
```
Comment (cwillum) [inline _dashboards/visualize/maps-plugin.md:32]: "... in the **Layers** panel, in the upper-left corner of the map, ..."
(suggestion only-feel free to disregard)

File: _dashboards/visualize/maps-plugin.md [hunk 5]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+- To add a map to a new dashboard, first create the dashboard as follows:
+
+  1. On the top menu bar, go to **OpenSearch Dashboards > Dashboard**.
+  1. Select the **Create dashboard** button.
+  1. Select the **Create new** button.
+
+- To add a map to an existing dashboard, first open the dashboard as follows:
+  1. On the top menu bar, go to **OpenSearch Dashboards > Dashboard**.
+  1. Select the dashboard you want to open from the list.
+  1. In the upper right corner, select **Edit**.
```
Comment (cwillum) [inline _dashboards/visualize/maps-plugin.md:185]: upper-right corner

File: _dashboards/visualize/maps-plugin.md [hunk 6]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+1. From the top menu, select **Add**.
+1. In the **Types** dropdown list, select **Maps**.
+1. Select the map you want to add from the list.
+
+### Adding a new map
+
+1. From the top menu, select the **Create new** button.
+1. In the **New Visualization** dialog, select **Maps**.
+1. Add any default map, layers, and tooltips to the map.
+1. In the upper right corner, select the **Save** button.
```
Comment (cwillum) [inline _dashboards/visualize/maps-plugin.md:200]: upper-right corner

File: _dashboards/visualize/maps-plugin.md [hunk 7]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+1. In the **Save map** dialog, enter the **Title** and optional **Description** of the map.
+1. Select **Add to Dashboard after saving**.
+1. Select the **Save and return** button.
+
+## Editing a map from a dashboard
+
+1. In the dashboard, select the gear icon in the upper right corner of the map you want to edit.
```
Comment (cwillum) [inline _dashboards/visualize/maps-plugin.md:207]: upper-right corner

File: _dashboards/visualize/maps-plugin.md [hunk 8]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+1. Select **Edit maps**.
+1. Edit the map.
+1. In the upper right corner, select the **Save** button.
```
Comment (cwillum) [inline _dashboards/visualize/maps-plugin.md:210]: upper-right corner

**Review by cwillum — APPROVED** [review]
Looks great. Steps are very clear to me.

**Review by ariamarble — APPROVED** [review]

File: _dashboards/visualize/maps-plugin.md (hunk 3 above)
Comment (kolchfa-aws) [reply _dashboards/visualize/maps-plugin.md:17]: Yes, they are actually menu items in the top menu... I don't know whether there is a better word than "workflow" here. Let's ask @natebower

File: _dashboards/visualize/maps.md [hunk 9]
```diff
@@ -1,22 +1,35 @@
...
-To get started, perform the following steps:
+You can create a new map from the **Maps** or **Visualize** workflows by performing the following steps: 
 
-1. On the top menu bar, go to **OpenSearch Plugins > Maps**.
-1. Select the **Create map** button. You can now see the default OpenSearch basemap.
-1. To examine the **Default map** layer configuration, in the **Layers** panel on the upper left of the map, select **Default map**, as shown in the following image.
+- To create a new map from the **Maps** workflow, perform the following steps:
+
+  1. On the top menu bar, go to **OpenSearch Plugins > Maps**. 
+  1. Select the **Create map** button.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:22]:
```suggestion
  1. Choose the **Create map** button. 
```

File: _dashboards/visualize/maps.md [hunk 10]
```diff
@@ -1,22 +1,35 @@
...
+- To create a new map from the **Maps** workflow, perform the following steps:
+
+  1. On the top menu bar, go to **OpenSearch Plugins > Maps**. 
+  1. Select the **Create map** button. 
+
+- To create a new map from the **Visualize** workflow, perform the following steps:
+
+  1. On the top menu bar, go to **OpenSearch Dashboards > Visualize**.
+  1. Select the **Create visualization** button.
+  1. In the **New Visualization** dialog, select **Maps**.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:28]: Global: Use "choose" to describe moving to a UI component such as a tab or pane or taking action on a button or menu.

File: _dashboards/visualize/maps.md [hunk 11]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+To open your saved map, select **Maps** in the upper-left corner. The list of saved maps is displayed.
+
+## Adding a map to a dashboard
+
+You can add a new or existing map to a new or existing dashboard by performing the following steps:
+
+- To add a map to a new dashboard, first create the dashboard as follows:
+
+  1. On the top menu bar, go to **OpenSearch Dashboards > Dashboard**.
+  1. Select the **Create dashboard** button.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:179]:
```suggestion
  1. Choose the **Create dashboard** button.
```

File: _dashboards/visualize/maps.md [hunk 12]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+  1. Select the **Create new** button.
+
+- To add a map to an existing dashboard, first open the dashboard as follows:
+  1. On the top menu bar, go to **OpenSearch Dashboards > Dashboard**.
+  1. Select the dashboard you want to open from the list.
+  1. In the upper-right corner, select **Edit**.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:185]:
```suggestion
  1. In the upper-right corner, choose **Edit**.
```

File: _dashboards/visualize/maps.md [hunk 13]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+
+### Adding an existing map
+
+1. From the top menu, select **Add**.
+1. In the **Types** dropdown list, select **Maps**.
+1. Select the map you want to add from the list.
+
+### Adding a new map
+
+1. From the top menu, select the **Create new** button.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:197]:
```suggestion
1. From the top menu, choose the **Create new** button.
```

File: _dashboards/visualize/maps.md [hunk 14]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+1. In the **New Visualization** dialog, select **Maps**.
+1. Edit the default map by adding a basemap, layers, or tooltips.
+1. In the upper-right corner, select the **Save** button.
+1. In the **Save map** dialog, enter the **Title** and optional **Description** of the map.
+1. Select **Add to Dashboard after saving** (this option is selected by default).
+1. Select the **Save and return** button.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:203]:
```suggestion
1. Choose the **Save and return** button.
```

File: _dashboards/visualize/maps.md [hunk 15]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+
+## Editing a map from a dashboard
+
+1. In the dashboard, select the gear icon in the upper-right corner of the map you want to edit.
+1. Select **Edit maps**.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:208]:
```suggestion
1. Choose **Edit maps**.
```

File: _dashboards/visualize/maps.md [hunk 16]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+1. In the **New Visualization** dialog, select **Maps**.
+1. Edit the default map by adding a basemap, layers, or tooltips.
+1. In the upper-right corner, select the **Save** button.
+1. In the **Save map** dialog, enter the **Title** and optional **Description** of the map.
+1. Select **Add to Dashboard after saving** (this option is selected by default).
+1. Select the **Save and return** button.
+
+## Editing a map from a dashboard
+
+1. In the dashboard, select the gear icon in the upper-right corner of the map you want to edit.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:207]:
```suggestion
1. In the dashboard, choose the gear icon in the upper-right corner of the map you want to edit.
```

File: _dashboards/visualize/maps.md [hunk 17]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+1. Select **Edit maps**.
+1. Edit the map.
+1. In the upper-right corner, select the **Save** button.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:210]:
```suggestion
1. In the upper-right corner, choose the **Save** button.
```

File: _dashboards/visualize/maps.md [hunk 18]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+1. In the **Save map** dialog, select the **Save and return** button.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:211]:
```suggestion
1. In the **Save map** dialog, choose the **Save and return** button.
```

File: _dashboards/visualize/maps.md [hunk 19]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
...
+### Adding an existing map
+
+1. From the top menu, select **Add**.
+1. In the **Types** dropdown list, select **Maps**.
+1. Select the map you want to add from the list.
+
+### Adding a new map
+
+1. From the top menu, select the **Create new** button.
+1. In the **New Visualization** dialog, select **Maps**.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:198]:
```suggestion
1. In the **New Visualization** dialog, choose **Maps**.
```

File: _dashboards/visualize/maps.md [hunk 20]
```diff
@@ -154,4 +167,45 @@ To save a map with all the layers that you set up, perform the following steps:
 1. (Optional) In the **Description** text box, enter the map description.
 1. Select the **Save** button.
 
-To open your saved map, select **Maps** in the upper-left corner. The list of saved maps is displayed.
\ No newline at end of file
+To open your saved map, select **Maps** in the upper-left corner. The list of saved maps is displayed.
```
Comment (natebower) [inline _dashboards/visualize/maps.md:170]:
```suggestion
To open your saved map, choose **Maps** in the upper-left corner. The list of saved maps is displayed.
```

File: _dashboards/visualize/maps.md (hunk 10 above)
Comment (natebower) [inline _dashboards/visualize/maps.md:28]:
```suggestion
  1. In the **New Visualization** dialog, choose **Maps**.
```

**Review by natebower — COMMENTED** [review]
@kolchfa-aws Just some easy word replacements. Let me know if you have any questions. Thanks!

**Pull request merged**